linera-core/src/client.rs

```diff
@@ -331,25 +331,24 @@ where
                 Ok(m.owner)
             }
             ChainManagerInfo::Multi(m) => {
```

Comment:
nit: Unrelated but `m` should be `manager` here to be consistent without guidelines (this may be on me).